### PR TITLE
docs: Update README

### PR DESCRIPTION
**Pull request by kayousterhout**
The existing README was out of date: it referenced SparkPI.js, which
doesn't exist anymore, and it also referenced old Kelda functionality
(e.g., like the need to set QUILT_PATH).  This commit updates the
README.

We probably want to add more, like how to start the spark-shell;
the goal of this PR is to make a minimal not-incorrect version.

### CODE REVIEW

**Comment by kayousterhout** [comment]
@ejj might make sense for you to review this since you've been doing the Spark stuff

File: README.md [hunk 1]
```diff
@@ -1,49 +1,36 @@
...
+
+```console
+$ git clone https://github.com/kelda/spark.git
+$ cd spark
+$ npm install .
+```
+
+## Launch an Apache Spark cluster
+
+To launch a Spark cluster, first start a Kelda daemon, if you don't already have
```
Comment (ejj) [inline README.md]: I wonder if this should instead/additionally link to the main kelda docs on how to run the daemon and what not.

File: README.md [hunk 2]
```diff
@@ -1,49 +1,36 @@
...
-[GettingStarted.md](https://github.com/NetSys/kelda/blob/master/docs/GettingStarted.md#set-up-your-ssh-authentication).
+The daemon is a long-running process, so you'll need to run the Kelda blueprint
+in a different window:
 
-### Build `sparkPI.js`
-Start the Kelda daemon with `kelda daemon`. Then, in a separate shell, execute
-`kelda run github.com/kelda/spark/sparkPI.js` to
-build this Kelda.js specification.
+```console
+$ kelda run ./sparkRun.js
```
Comment (ejj) [inline README.md]: Should we suggest they run `kelda init`?  This won't work if they haven't.

**Comment by kayousterhout** [comment]
@ejj updated

**Pull request merged**